selectors-4/Overview.bs

```diff
@@ -2813,52 +2813,12 @@ The Optionality Pseudo-classes: '':required'' and '':optional''</h4>
 <h4 id="user-pseudos">
 The User-interaction Pseudo-classes: '':user-valid'' and '':user-invalid''</h4>
 
-	The <dfn id="user-invalid-pseudo">:user-invalid</dfn>
-	and the <dfn id="user-valid-pseudo">:user-valid</dfn> pseudo-classes
-	represent an element with incorrect or correct input, respectively,
-	but only <em>after</em> the user has significantly interacted with it.
-
-	The '':user-invalid'' pseudo-class
-	must match an '':invalid'', '':out-of-range'', or
-	blank-but-'':required'' elements between the time the user has attempted
-	to submit the form and before the user has interacted again with the
-	form element.
-
-	The '':user-valid'' pseudo-class must match a '':valid'' element between the
-	time the user has attempted to submit the form and before the user has
-	interacted again with the form element.
-
-	User-agents may allow them to match such elements at other times,
-	as would be appropriate for highlighting an error to the user.
-	For example, a UA may choose to have '':user-invalid'' match an
-	'':invalid'' element once the user has typed some text into it and
-	changed the focus to another element, and to stop matching only after
-	the user has successfully corrected the input.
-
-	<div class='example'>
-		For example, the input in the following document fragment
-		would match '':invalid'' as soon as the page is loaded
-		(because it the initial value violates the max-constraint),
-		but it won't match '':user-invalid'' until the user significantly interacts with the element,
-		or attempts to submit the form it's part of.
-
-		<pre>
-		&lt;form>
-		  &lt;label>
-		    Volume:
-		    &lt;input name='vol' type=number min=0 max=10 value=11>
-		  &lt;/label>
-		  ...
-		&lt;/form>
-		</pre>
-	</div>
+  The '':user-valid'' and '':user-invalid'' pseudo-classes are defined in HTML here: https://html.spec.whatwg.org/#selector-user-valid
```

Comment:
I think I'd probably do `<a href="https://html.spec.whatwg.org/#selector-user-valid">in HTML</a>.` or so (or using some bikeshed magic to get that), rather than dropping the raw URL there.

@tabatkins or @fantasai should look at this as editors, but lgtm for the general change, we shouldn't have two incompatible definitions of these.

Reply:
I suspect we'll want language like "are defined by the host language" along with a vague description of what the selectors are about.

Reply:
Ok, I brought back the summary paragraph and said that its defined by the host language instead of html